### PR TITLE
xbytes2: Improve Tracker Name, WEBDL Detection, Languages. Freeleench only option

### PR DESCRIPTION
**Pull request by danimart1991**
- Add tracker Name to the end of filename
- Improve WebDL detection
- Improve French language detection
- Option to search only Freeleench downloads.

### CODE REVIEW

File: src/Jackett.Common/Definitions/xbytes2.yml [hunk 1]
```diff
@@ -114,6 +121,12 @@ search:
     title:
       selector: td[valign="middle"] a
       filters:
+        - name: re_replace
+          args: ["\\W", "."] # Spaces and other characters -> .
```
Comment (ngosang) [inline src/Jackett.Common/Definitions/xbytes2.yml:125]: Replacing spaces with dots is not necessary. Sonarr/Radarr works well with spaces and the text is more readable.

File: src/Jackett.Common/Definitions/xbytes2.yml [hunk 2]
```diff
@@ -143,6 +156,18 @@ search:
...
+        - name: re_replace
+          args: ["WEB.DL", "WEBDL"] # Fix for Radarr
+        - name: re_replace
+          args: ["WEBDL.MHD.BDRip", "WEBDL"] # Fix for Radarr
+        - name: re_replace
+          args: ["WEBRIP.MHD.BDRip", "WEBRIP"] # Fix for Radarr
+        - name: append
+          args: "-XbytesV2"
+        - name: re_replace
+          args: ["\\.-XbytesV2", "-XbytesV2"] # Delete first dot
```
Comment (ngosang) [inline src/Jackett.Common/Definitions/xbytes2.yml:170]: Appending tracker name is not required and we are not doing it in other trackers. In Sonarr/Radarr you can configure tracker priority if you are trying to prefer xbytes2 results or something like that.

**Review by ngosang — CHANGES_REQUESTED** [review]

File: src/Jackett.Common/Definitions/xbytes2.yml (hunk 2 above)
Comment (danimart1991) [reply src/Jackett.Common/Definitions/xbytes2.yml]: Removed

File: src/Jackett.Common/Definitions/xbytes2.yml (hunk 1 above)
Comment (danimart1991) [reply src/Jackett.Common/Definitions/xbytes2.yml]: Removed.

The another two dots related replace fix issues on naming.

**Review by ngosang — APPROVED** [review]

**Comment by ngosang** [comment]
Thank you!

**Pull request merged**